sympy/vector/coordsys.py

```diff
@@ -44,6 +54,7 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             Used for simple str printing.
 
         """
+        from sympy.abc import a, x, y, z
```

Comment:
Moreover, since these are used to denote angles etc. in some coordinate systems, more 'neutral' names could be better. For example, u, v, w instead of x, y, z.